google/oauth2/_id_token_async.py

```diff
@@ -93,9 +93,9 @@ async def _fetch_certs(request, certs_url):
             "Could not fetch certificates at {}".format(certs_url)
         )
 
-    data = await response.data.read()
+    data = await response.content()
 
-    return json.loads(json.dumps(data))
+    return json.loads(data)
```

Comment:
@crwilcox Can you comment on this implementation?  The non-async version just does:

```python
return json.loads(response.data.decode("utf-8"))
```

I'm surprised that the synchronous version, at least, does not use `response.json`.

Reply:
I run into the same error.
Async library does not perform automatic decompression.
response.data.read() returns compressed bytes.
response.content() returns decompressed data as string